corehq/apps/sso/tasks.py

```diff
@@ -99,3 +109,53 @@ def send_idp_cert_expires_reminder_emails(num_days):
                 f"Failed to send cert reminder email for IdP {idp}: {exc!s}",
                 exc_info=True,
             )
+
+
+@periodic_task(run_every=crontab(minute=0, hour=2), acks_late=True)
+def auto_deactivate_removed_sso_users():
+    for idp in IdentityProvider.objects.filter(
+        enable_user_deactivation=True,
+        idp_type=IdentityProviderType.AZURE_AD
+    ).all():
+        # Fetch a list of users usernames that are members of the idp
```

Comment:
```suggestion
        # Fetch a list of users usernames that are members of domains belonging to the Identity Provider's Billing Account owner
```

Reply:
can the resulting list be its own utility?

Reply:
Hi Biyeun, this function will get all users of the idp, even though some of them might not be our hq web user

Reply:
I would say that this comment isn't necessary here. The variable name and method name should be descriptive enough, and if someone wants more of an explanation, they can look at the docstring for the method  `get_all_members_of_the_idp`. So I'd suggest moving this comment to a docstring on the method.

Reply:
Go with Graham's suggestion in  9e871028b9474127c75e96287df074446cfd57ce